forge-gui/res/cardsfolder/d/devout_invocation.txt

```diff
@@ -4,7 +4,7 @@ Types:Sorcery
 A:SP$ Tap | CardChoices$ Creature.YouCtrl+untapped | AnyNumber$ True | ChoiceAmount$ Count$Valid Creature.YouCtrl+untapped | RememberTapped$ True | SubAbility$ DBToken | SpellDescription$ Tap any number of untapped creatures you control. Create a 4/4 white Angel creature token with flying for each creature tapped this way. | StackDescription$ SpellDescription
 SVar:DBToken:DB$ Token | TokenAmount$ Y | TokenScript$ w_4_4_angel_flying | TokenOwner$ You | SubAbility$ DBCleanup
 SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
-SVar:Y:Remembered$Amount
+SVar:Y:Count$RememberedSize
```

Comment:
This should not lead to any other results

The card works fine for me, did you test this in a clean setting (e.g. only basics in all zones)?

Reply:
I tested 1v1 vs ai with a deck that was all land + fast mana + 0 mana creatures
See in the log: striker attacks for 4 on previous turn, this turn full tap but no tokens
![image](https://github.com/Card-Forge/forge/assets/3029794/993daa40-2d48-412a-b1f5-b55ba46ab96d)

```
Deck: test

Main: 
4 Black Lotus
4 Crimson Kobolds
4 Crookshank Kobolds
4 Devout Invocation
4 Kobolds of Kher Keep
4 Mana Crypt
4 Mox Pearl
4 Ornithopter
20 Plains
4 Siege Striker
4 Sol Ring
```

Reply:
I'm on version 1.6.56

Reply:
In that case you need to compare your local version to the repository

You'll notice it already got a fix after that release was done

Reply:
Gotcha, thanks!